src/xcode/ENA/ENA/Source/Scenes/FriendsInvite/FriendsInviteController.swift

```diff
@@ -79,11 +79,15 @@ final class FriendsInviteController: UIViewController {
 
 extension FriendsInviteController: UIActivityItemSource {
 	func activityViewControllerPlaceholderItem(_: UIActivityViewController) -> Any {
-		shareTitle
+		return shareTitle
 	}
 
-	func activityViewController(_: UIActivityViewController, itemForActivityType _: UIActivity.ActivityType?) -> Any? {
-		shareUrl
+	func activityViewController(_: UIActivityViewController, itemForActivityType activityType: UIActivity.ActivityType?) -> Any? {
+		if activityType == .mail {
+			return ""
```

Comment:
if you return a nil, what will happen?

Reply:
The subject is gone from Apple Mail app. :)